### ISSUE TITLE
iOS Client biometrics wants to use fingerprint instead of face unlock on iPhone 11

### ISSUE THREAD

**Issue by michelinux** [bitwarden/mobile#3204]
### Steps To Reproduce

1. Open BitWarden app
2. Login
3. At the "Verify master password" screen, try to use biometrics
4. Only button available is "Use fingerprint to unlock", while it was expecting "Use Face ID To Unlock".
5. Click on "Use fingerprint to unlock", doesn't work as expected as iPhone 11 does not have fingerprint. 

### Expected Result

Expecting to see "Use Face ID To Unlock" button when unlocking the vault.

### Actual Result

"Use fingerprint to unlock" is the biometric proposed by the app.

### Screenshots or Videos

![IMG_91E1CFAF4E3B-1](https://github.com/bitwarden/mobile/assets/1716189/47a71579-fc46-47f8-be37-0e96ab4f8461)

### Additional Context

I has been working with Face ID for a long time, at least until a few days before 28 April. Current version: 2024.4.0 (7272).
iPhone 11 Pro Max
iOs Version: 17.4.1

### Operating System

iOS

### Operating System Version

17.4.1

### Device

iPhone 11 Pro Max

### Build Version

2024.4.0 (7272)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by SergeantConfused** [comment]
Hello @michelinux,

Thank you for reporting this. I recommend that you proceed with a fresh new installation, as I am unable to reproduce this and get that particular button to show up; Please uninstall Bitwarden off your iOS device, turn the phone off and back on, and install Bitwarden anew via the Apple App Store.

If this behaviour persists, please feel free to create a [Support ticket](https://bitwarden.com/help/) so we'd have a look at this together, and please also include a link to this GitHub report in the body of your ticket.

Alternatively, you can seek assistance from other Bitwarden users in our [Community Forums](https://community.bitwarden.com/c/support/), if you wish.

We use GitHub to track bugs and other development related matters; This GitHub report will be closed at this point.

Thank you again,

**Comment by michelinux** [comment]
Didn't solve with a reinstall. I had to reinstall and restart of the phone, which did solve the issue.

I strongly suspect that the issue originated from me moving from an Asian country to Europe, and the iOS applying the new EU rules. This could be a side effect of the adaptations iOS has to comply when in EU.

I have added this note for future reference. I agree this is not a bug in Bitwarden, rather a bug in iOS. 

Many thanks.
